twitter/twitter.php

```diff
@@ -1194,6 +1197,13 @@ function twitter_do_mirrorpost(App $a, int $uid, $post)
 	return $datarray;
 }
 
+/**
+ * Fetches the Twitter user's own posts
+ *
+ * @param App $a
+ * @param int $uid
```

Comment:
I would not mind having an `@return void` being added here, later on I will add `: void` for methods returning nothing to be ready for newer PHP versions.

Reply:
Done.